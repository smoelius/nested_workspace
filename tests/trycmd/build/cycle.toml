args = ["build", "-vv", "--offline"]

status = "failed"

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/cycle"
