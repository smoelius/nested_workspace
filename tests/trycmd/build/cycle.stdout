[cycle 0.1.0] [..]
